src/classes/content/wordpress/class-abstract-wordpress-content-service.php

```diff
@@ -69,13 +70,24 @@ public function get_about_jsonld( $content_id ) {
 
 	/**
 	 * @param Wordpress_Content_Id $content_id
-	 * @param string               $value
+	 * @param string $value
+	 *
+	 * @throws Exception If the 'match_name' column does not exist in the database table.
 	 */
 	public function set_about_jsonld( $content_id, $value ) {
 		global $wpdb;
 
 		// Cleanup value.
-		$value = ( is_string( $value ) && strlen( $value ) > 2 ) ? $value : null;
+		$value      = ( is_string( $value ) && strlen( $value ) > 2 ) ? $value : null;
+		$match_name = "NULL";
+
+		if ( $value ) {
+			// Check if the 'match_name' column exists in the database table
+			$columns = $wpdb->get_col_info( 'name', 0 );
+			if ( in_array( 'match_name', $columns ) ) {
+				$match_name = $this->get_match_name( $value );
```

Comment:
I see that `get_match_name` is written 3 times, maybe it's best to create a Utils class, or an About_Jsonld class or an abstract class so that we avoid duplicating code?